### DISCUSSION TITLE
Horizontal scheduler calendar

### DISCUSSION THREAD

**Discussion by daramousk** [ant-design/ant-design#42684] [Ideas]
Hi,

Thanks for this amazing library, just testing it out for a few projects and working great!
The only thing I cannot find though, is something like this https://fullcalendar.io/docs/timeline-view

Is there perhaps a prop that I am missing or is there something planned that I can contribute to?

Thank you.

**Comment by bombillazo** [comment]
Hey! This is not implemented in the current calendar component. But the Ant Design team is open for contributions, usually, you can start an RFC discussion here and the core team can give feedback and help out.
